### DISCUSSION TITLE
Static cache pages - with forms

### DISCUSSION THREAD

**Discussion by Kanderka** [statamic/cms#4377] [Q&A]
If I am using static pages full strategy and Almost on all pages I have call to action form, Is there some way how to make static page without 419 form error? I know that I can exclude each page with form from caching but it's not so fast.
Is there some way how to do it?

**Comment by jasonvarga** [comment]
You can use JavaScript to get the csrf token and submit the form.

The "Peak" starter kit uses this technique. https://peak.studio1902.nl/features/forms.html

**Reply by jackmcdade** [reply]
There's a bit in the docs about this too: https://statamic.dev/forms#using-ajax
